ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -1385,6 +1413,12 @@ object JdbcLedgerDao {
         |from parameters
         |on conflict (package_id) do nothing""".stripMargin
 
+    override protected[JdbcLedgerDao] val SQL_INSERT_PACKAGE_REJECTS: String =
+      """insert into package_rejects(package_id, participant_id, submission_id, reason)
```

Comment:
I would follow the approach that we're doing with transactions: we have one table for per-submission kind that records both accepts and rejects.

Reply:
went with separate table